src/v1/examples/ERC20Distributor.sol

```diff
@@ -90,11 +91,12 @@ contract ERC20Distributor is MockERC20, LPNClientV1 {
             ? min(block.number, CAMPAIGN_END_BLOCK) - 1
             : endBlock_;
 
+        lastClaimed[holder] = endBlock;
+
         uint256 requestId = lpnRegistry.request{value: lpnRegistry.gasFee()}(
             queryHash, placeholders, startBlock, endBlock
         );
 
-        lastClaimed[holder] = endBlock;
         requestData[requestId] =
             RequestData({holder: holder, blockRange: endBlock - startBlock + 1});
     }
```

Comment:
change is made to enforce the "checks-effects-interactions" patter (which fixes a reentrancy detection)